nsdb-cluster/src/main/resources/cluster.conf

```diff
@@ -118,6 +118,15 @@ nsdb {
     api.version = "v0.1"
   }
 
+  ui {
```

Comment:
May we add a prop based on which serve or not NSDb ui ?

Reply:
good point. I'll address it.